pkgs/by-name/cl/clash-verge-rev/package.nix

```
@@ -1,19 +1,132 @@
-{ lib
-, clash-verge
-, fetchurl
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  nodejs,
+  pnpm,
+  cargo,
+  rustc,
+  rustPlatform,
+  cargo-tauri,
+  pkg-config,
+  wrapGAppsHook3,
+  glib,
+  gtk3,
+  libsoup,
+  openssl,
+  webkitgtk,
+  libayatana-appindicator,
+  mihomo,
+  dbip-country-lite,
+  v2ray-geoip,
+  v2ray-domain-list-community,
 }:
+let
+  mihomo-alpha = mihomo.overrideAttrs (
+    finalAttrs: previousAttrs: {
```

```suggestion
    {
```
I think they are safe to remove. Don't forget about the quotes.